tools/bundle/src/index.css

```diff
@@ -0,0 +1,99 @@
+/*!
+ * Copyright 2024 Adobe. All rights reserved.
+ *
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at <http://www.apache.org/licenses/LICENSE-2.0>
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
```

Comment:
Would a comment saying this is generated be at all useful to help guide folks in the future if there needs to be changes or bug fixes?

Reply:
Oh great idea!  Yes, I'll add that.

Reply:
@5t3ph pushed that up just now